### ISSUE TITLE
Validation error message remains visible when moving through transitions

### ISSUE THREAD

**Issue by yosefmah** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#1969]
Posted by @ChrisCross67 [here](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/pull/1842#issuecomment-653526645)

![image](https://user-images.githubusercontent.com/8480749/86469597-0356e280-bd3a-11ea-939e-24d052032796.gif)

I'm not sure where the problem lies, wrapping the entire control in an AdornerDecorator fixes it but would be good to sort it in the template.  I've tried but with no success.

Here is the sample XAML:

```
<materialDesign:Transitioner SelectedIndex="0">
        <StackPanel HorizontalAlignment="Center">
            <TextBlock Text="First slide" />
            <TextBox materialDesign:HintAssist.HelperText="Helper text longer than error" materialDesign:HintAssist.Hint="Name">
                <TextBox.Text>
                    <Binding Path="Name" UpdateSourceTrigger="PropertyChanged">
                        <Binding.ValidationRules>
                            <domain1:NotEmptyValidationRule ValidatesOnTargetUpdated="True" />
                        </Binding.ValidationRules>
                    </Binding>
                </TextBox.Text>
            </TextBox>

            <Button Command="{x:Static materialDesign:Transitioner.MoveNextCommand}"
                    Style="{DynamicResource MaterialDesignFloatingActionButton}">
                <materialDesign:PackIcon Kind="ArrowRight" />
            </Button>
        </StackPanel>
        <StackPanel HorizontalAlignment="Center">
            <TextBlock Text="Second slide" />
            <Button Command="{x:Static materialDesign:Transitioner.MovePreviousCommand}"
                    Style="{DynamicResource MaterialDesignFloatingActionButton}">
                <materialDesign:PackIcon Kind="ArrowLeft" />
            </Button>
        </StackPanel>
    </materialDesign:Transitioner>
```

**Comment by Keboo** [comment]
I believe the issue here is that each slide is the transitioner should likely be getting its own Adorner layer. I suspect this is likely solved in the 3.2.0 _preview_ NuGet packages as we added an adorner layer to to the TextBoxes so those should likely be handled now.

**Comment by ChrisCross67** [comment]
Thanks @Keboo, I tried with the 3.2.0 preview, same result...
I think the problem is related to ValidationErrorTemplate. I will do some test and let you know ;) 

![MuwGr3NFdn](https://user-images.githubusercontent.com/8480749/87125581-05b1c300-c28b-11ea-9f7f-005214d4321f.gif)
